src/layouts/partials/footer.html

```diff
@@ -12,5 +14,39 @@
     <script type="text/javascript" src="/js/aside.js?{{ partial "cachebreaker_js.html" . }}"></script>
     <script type="text/javascript" src="/js/base.js?{{ partial "cachebreaker_js.html" . }}"></script>
 
+    <script>
+      // Recreate open/close behaviour of the mobile menu.
+      var mobileMenuTrigger = document.querySelector('.menu-trigger__container');
+      var mobileMenu = document.querySelector('.mobile-menu');
+      mobileMenuTrigger.addEventListener('click', () => {
+        mobileMenuTrigger.classList.toggle('open');
+        mobileMenu.classList.toggle('open');
+      });
+
+      // Recreate open/close behaviour of individual menu items in the mobile menu.
+      var mobileMenuLinks = document.querySelectorAll('.mobile-menu .hs-menu-depth-1 > a');
+      for (let i = 0; i < mobileMenuLinks.length; i++) {
+        let link = mobileMenuLinks[i];
+
+        link.addEventListener('click', (e) => {
+          e.preventDefault();
+          let ex = link.getAttribute('aria-expanded');
+          if (ex === 'true') {
+            link.setAttribute('aria-expanded', 'false')
+          } else {
+            link.setAttribute('aria-expanded', 'true')
+          }
+
+          let itemContainer = link.nextElementSibling.querySelector('.hs-menu-children-wrapper');
```

Comment:
```suggestion
        var link = mobileMenuLinks[i];

        link.addEventListener('click', (e) => {
          e.preventDefault();
          var ex = link.getAttribute('aria-expanded');
          if (ex === 'true') {
            link.setAttribute('aria-expanded', 'false')
          } else {
            link.setAttribute('aria-expanded', 'true')
          }

          var itemContainer = link.nextElementSibling.querySelector('.hs-menu-children-wrapper');
```

Reply:
Had to do something a little bit different there since the addEventListener is a function which gets executed after the for loop is done, at which point `link` will be the last link in the list. Without `let` that means all links open the menu one instead of the one they belong to.

So I made a function that bakes in the value of `link` in that iteration and returns a click handler instead.